contracts/VM.sol

```diff
@@ -89,9 +80,8 @@ abstract contract VM {
                 }(
                     // inputs
                     state.buildInputs(
-                        //selector
                         bytes4(command),
-                        bytes32(uint256(indices << 8) | CommandBuilder.IDX_END_OF_ARGS)
```

Comment:
can you explain what removing this flag accomplishes so that I understand?

Reply:
how will "the END_OF_ARGS will already be present". The left shift exposes a bytes1(0). The END_OF_ARGS is 0xff.

Reply:
indices is a list of 1 bytes as defined here https://github.com/weiroll/weiroll#inputoutput-list-ino-format and afaik has to end with 0xff to indicate end of args so i assume it will be present even after shifting by 1 byte (as so to remove the value input byte which will always be encoded as the first input)